public/robots.public.txt

```diff
@@ -36,3 +34,6 @@ Disallow: /
 
 User-agent: Slurp
 Crawl-delay: 30
+
```

Comment:
Based on [this discussion](https://github.com/otwcode/otwarchive/pull/4411#discussion_r1044351129), I think this should come before the user agent with the `crawl-delay`.